src/SizeMeApp.jsx

```diff
@@ -94,6 +96,16 @@ class SizeMeApp extends React.Component {
                             {loggedIn && <ProfileMenu profiles={profiles}
                                 selectedProfile={selectedProfile.id}
                                 setSelectedProfile={setSelectedProfile}/>}
+                            {!loggedIn && (<div className="profile-menu-container">
+                                <img src={sizemeIcon} alt="SizeMe" data-tip data-for="sizeme-tooltip" data-event="click"
+                                    ref={el => {this.tooltip = el;}} onMouseEnter={() => ReactTooltip.show(this.tooltip)}/>
```

Comment:
I don't know if the hover-tooltip is good or annoying. If latter, remove `onMouseEnter`-function here and only click will open the tooltip